modules/040-node-manager/images/capi-controller-manager/werf.inc.yaml

```diff
@@ -0,0 +1,27 @@
+{{- $version := "1.4.4" }}
```

Comment:
How do you think, could we bump that to latest 1.5.2 ?

Reply:
done